app/lib/actions/candlepin/product/update.rb

```diff
@@ -18,6 +20,7 @@ def plan(product)
                                          :label => pc.content.label,
```

Comment:
Should this call to ::Actions::Candlepin::Product::ContentCreate include the owner?